src/licensedcode/data/licenses/hpnd-netrek.LICENSE

```diff
@@ -0,0 +1,17 @@
+---
+key: hpnd-netrek
+short_name: HPND Netrek variant
+name: Historical Permission Notice and Disclaimer Netrek variant
+category: Permissive
+owner: S. M. Patel
+spdx_license_key: HPND-Netrek
+---
+
+Permission to use, copy, modify, and distribute this
```

Comment:
https://github.com/aboutcode-org/scancode-toolkit/blob/develop/src/licensedcode/data/rules/mit-old-style_13.RULE is mighty similar! ... we need to keep at least an issue to consider if this should be requalified later.

Reply:
https://github.com/aboutcode-org/scancode-toolkit/issues/3898, thanks!